### PR TITLE
Cardview

### PR DESCRIPTION
**Pull request by mavalo99**

### CODE REVIEW

**Comment by raphaelseher** [comment]
|File|Coverage [4.39%]|:green_apple:|
|:-|:-:|:-:|
|[GameField.java](https://github.com/SE2-Gwent/Gwent/blob/30f0e9899f194b95bd58de2224f70e499600770e/gamelogic%2Fsrc%2Fmain%2Fjava%2Fat%2Faau%2Fse2%2Fgamelogic%2Fmodels%2FGameField.java)|65.85%|:green_apple:|
|[Hero.java](https://github.com/SE2-Gwent/Gwent/blob/30f0e9899f194b95bd58de2224f70e499600770e/gamelogic%2Fsrc%2Fmain%2Fjava%2Fat%2Faau%2Fse2%2Fgamelogic%2Fmodels%2FHero.java)|0%|:green_apple:|
|[PlayingCardDetailed.java](https://github.com/SE2-Gwent/Gwent/blob/30f0e9899f194b95bd58de2224f70e499600770e/app%2Fsrc%2Fmain%2Fjava%2Fat%2Faau%2Fse2%2Fgwent%2FPlayingCardDetailed.java)|0%|:green_apple:|
|[StartFragment.java](https://github.com/SE2-Gwent/Gwent/blob/30f0e9899f194b95bd58de2224f70e499600770e/app%2Fsrc%2Fmain%2Fjava%2Fat%2Faau%2Fse2%2Fgwent%2FStartFragment.java)|0%|:green_apple:|
|[MainActivity.java](https://github.com/SE2-Gwent/Gwent/blob/30f0e9899f194b95bd58de2224f70e499600770e/app%2Fsrc%2Fmain%2Fjava%2Fat%2Faau%2Fse2%2Fgwent%2FMainActivity.java)|0%|:green_apple:|
|[CardView.java](https://github.com/SE2-Gwent/Gwent/blob/30f0e9899f194b95bd58de2224f70e499600770e/app%2Fsrc%2Fmain%2Fjava%2Fat%2Faau%2Fse2%2Fgwent%2Fviews%2Fcommon%2FCardView.java)|0%|:green_apple:|

|Total Project Coverage|23.76%|:green_apple:|
|:-|:-:|:-:|

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,10 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/Default_Background_Detailed"
```
Comment (raphaelseher) [inline app/src/main/res/layout/activity_main.xml:6]: Obsolete?

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    android:soundEffectsEnabled="true"
+    android:splitMotionEvents="true"
+    android:visibility="visible"
```
Comment (raphaelseher) [inline app/src/main/res/layout/activity_main.xml:11]: Obsolete?

File: app/src/main/java/at/aau/se2/gwent/PlayingCardDetailed.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package at.aau.se2.gwent;
+
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+
+public class PlayingCardDetailed extends AppCompatActivity {
```
Comment (raphaelseher) [inline app/src/main/java/at/aau/se2/gwent/PlayingCardDetailed.java:6]: DetailCardView ist gemerged, diese Activity löschen und mit der von Kevin ersetzen, so wie hier aufrufen:
https://github.com/SE2-Gwent/Gwent/pull/46/files#diff-894fc26df703733daa3819363b76599837c45a936806b541016698daa485633bR97

**Review by raphaelseher — CHANGES_REQUESTED** [review]

File: app/src/main/java/at/aau/se2/gwent/PlayingCardDetailed.java (hunk 3 above)
Comment (mavalo99) [reply app/src/main/java/at/aau/se2/gwent/PlayingCardDetailed.java:6]: @raphaelseher 
Okay ich entferne die ganze Klasse. Soll auch die dazu gehörige XML raus?

**Pull request merged**